radiomics/firstorder.py

```diff
@@ -30,6 +30,12 @@ def __init__(self, inputImage, inputMask, **kwargs):
     self.pixelSpacing = inputImage.GetSpacing()
     self.voxelArrayShift = kwargs.get('voxelArrayShift', 0)
 
+    self._initLesionWiseCalculation()
+
+  def _initLesionWiseCalculation(self):
+    super(RadiomicsFirstOrder, self)._initLesionWiseCalculation()
+    self.targetVoxelArray = self.imageArray[self.ROICoordinates].astype('float')
+
     self.logger.debug('Feature class initialized')
```

Comment:
I'll add it, but in the log the logger is already feature class specific, (i.e. logger would be `radiomics.firstorder`). The reason I'll still add it is because the logger name is not necessarily printed (as for example in the standard output to the console when verbosity is high enough.